### PR TITLE
fix(Communities): Fix import community without chats

### PR DESCRIPTION
**Pull request by borismelnik**
Closes: #5818

### What does the PR do

Add possibility to import community without chats
should be merged after [status-go PR](https://github.com/status-im/status-go/pull/2694). Otherwise, after import app stuck on login.

### Affected areas

Communities

### CODE REVIEW

File: src/app_service/service/community/service.nim [hunk 1]
```diff
@@ -884,19 +884,17 @@ QtObject:
       if(communitiesSettingsJArr.len == 0):
         raise newException(RpcException, fmt"`communitiesSettings` array is empty in the response for community id: {communityKey}")
 
-      var chatsJArr: JsonNode
-      if(not response.result.getProp("chats", chatsJArr)):
-        raise newException(RpcException, fmt"there is no `chats` key in the response for community id: {communityKey}")
-
```
Comment (PascalPrecht) [inline src/app_service/service/community/service.nim:890]: Question:

In https://github.com/status-im/status-go/pull/2694#pullrequestreview-989994420 we expect that there's a `chats` key in the response. Don't we still want to keep this exception then?

**Review by PascalPrecht — APPROVED** [review]

File: src/app_service/service/community/service.nim (hunk 1 above)
Comment (borismelnik) [reply src/app_service/service/community/service.nim:890]: @PascalPrecht no as I understand we should be able to import community without chats.

**Review by richard-ramos — APPROVED** [review]

**Review by endulab — APPROVED** [review]

**Comment by elina2015** [comment]
@borismelnik now the community is imported, but without the name:
image
Could you kindly check?

**Review by osmaczko — APPROVED** [review]

**Comment by borismelnik** [comment]
@elina2015 seems like status-go wasn't able to retrieve its description, @richard-ramos  can tell more :)

**Pull request merged**